src/apis/vaults.ts

```diff
@@ -70,8 +123,10 @@ export class DefaultVaultsAPI {
     async selectRandomVaultIssue(btc: PolkaBTC): Promise<AccountId> {
         const customAPIRPC = this.api.rpc as any;
         try {
-            const firstVaultWithSufficientCollateral =
-                await customAPIRPC.vaultRegistry.getFirstVaultWithSufficientCollateral(btc);
+            // eslint-disable-next-line max-len
+            const firstVaultWithSufficientCollateral = await customAPIRPC.vaultRegistry.getFirstVaultWithSufficientCollateral(
```

Comment:
I did not catch this before but why do we have a try/catch? If we throw in an `async` function it will reject the promise, so this is redundant I think.